boards/aerotenna/ocpoc/src/CMakeLists.txt

```diff
@@ -31,6 +31,6 @@
 #
 ############################################################################
 
-px4_add_library(drivers_board
+add_library(drivers_board
 	${PX4_SOURCE_DIR}/src/drivers/boards/common/board_identity.c # TODO: this is horrible and should be fixed
```

Comment:
what is horrible and what need to be fixed

Reply:
Cleanup along the way. Moved to https://github.com/PX4/Firmware/pull/12446.